Bot.DSharp/DSharpClient.cs

```diff
@@ -40,19 +41,38 @@ public async Task ConnectAsync(IServiceProvider botServices)
         {
             var slash = m_discord.UseSlashCommands(new SlashCommandsConfiguration { Services = botServices });
 
-            // During development, register our commands to the dev guild only
-            List<ulong> devGuildIds = new() { 128585855097896963ul, 215551375608643586ul };
+            // If somehow neither deploy type is specified, don't screw around with anything
+            string deployType = Environment.GetEnvironmentVariable("DEPLOY_TYPE") ?? "none";
```

Comment:
I think this should be an exception that halts the bot entirely. That way if we run locally and forget to update the config file, we don't get confused as to why our commands aren't working.

Also, I think it should be up higher, above the `var slash =` line.

Reply:
Done, except the exception still occurs in roughly this spot